typing/typedecl.ml

```diff
@@ -403,7 +403,7 @@ let transl_declaration env sdecl (id, uid) =
         type_kind = kind;
         type_private = sdecl.ptype_private;
         type_manifest = man;
-        type_variance = Variance.unknown_signature ~arity;
+        type_variance = Variance.unknown_signature ~injective:false ~arity;
```

Comment:
Not sure it matters, but it seems like you could fill this one in based on the kind

Reply:
Actually, this information is never used, as the variance is computed in the next pass.
So I chose to keep the original default here.